src/test/resources/META-INF/resources/frontend/styles/granitealert/demo-styles.css

```
@@ -33,13 +33,36 @@ granite-alert-mixin.funny::part(alert) {
+
+granite-alert-mixin span {
+    padding-right: 20px;
+    display: block;
+}
+
 granite-alert-mixin vaadin-button {
 	background-color: transparent;
+	--vaadin-button-background: transparent;
 	float: right;
 	cursor: pointer;
     height: 26px;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "height" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)